montecarlo/configs/params.yaml

```diff
@@ -108,4 +105,53 @@ pointing_mode: "spin-stabilized" # "detumble" # "detumble" or "spin-stabilized"
 pointing_target: "Nadir" # "Nadir" or "Sun"
 tgt_ss_ang_vel: 10 # [deg/s]
 controller_dt: 1 # [s]
-estimator_dt: 1    # [s]
\ No newline at end of file
+estimator_dt: 1    # [s]
+
+# -----------------------------------------------------------------------------------------------------------
+# PARAMETER DISPERSION
+# -----------------------------------------------------------------------------------------------------------
+
+# Mass
+min_mass : 0.5 # Min allowable mass [kg]
+mass_dev : 50 # Stddev from nominal mass as a % of nominal mass [%]
+max_mass : 2.0 # Max allowable mass [kg]
+
+# Area
+min_area : 0.01 # Min area [m^2]
+area_dev : 20 # stddev from nominal area as a % of nominal area [%]
+
+# Inertia
+principal_axis_dev : [20,20,20] # stddev of principal axis inertias about nominal as a % of nominal [%]
+
+# Reaction wheels
```

Comment:
with this much uncertainty spin stabilization wouldnt work, more like 2% or less. we can fix this on a later pr

Reply:
Yea, I just set all of them to 20% pending further quantification of errors